main/vstring.c

```diff
@@ -81,12 +81,12 @@ extern vString *vStringNew (void)
 
 extern void vStringPut (vString *const string, const int c)
```

Comment:
@t-b I think until VS 2013 it didn't support C99, so it might be better to use `__inline` which AFAIK works on most compilers (MSVC, GCC, Clang, ICC, etc). Another option is to define some macros perhaps like (untested):

```c
#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L
# define CTAGS_INLINE static inline
#elif defined(_MSC_VER)
# define CTAGS_INLINE static __inline
#elif defined(__GNUC__) || defined(__clang__)
# define CTAGS_INLINE static __inline__
// #elif ... other compilers/tests here ...
// # define CTAGS_INLINE ...
#else
# define CTAGS_INLINE static
#endif
```